pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -55,6 +56,7 @@ stdenv.mkDerivation rec {
     ++ optionals numaSupport [ numactl ]
     ++ optionals pulseSupport [ libpulseaudio ]
     ++ optionals sdlSupport [ SDL2 ]
+    ++ optionals gtkSupport [ gtk3 gettext gnome3.vte-ng ]
```

Comment:
Is not standard `vte` enough? `vte-ng` is a fork that exposes some internal APIs.

Reply:
Thanks.  Changed to standard `vte`,  doesn't seem to make a difference.